opencti-platform/opencti-front/src/private/components/workspaces/dashboards/WidgetConfig.jsx

```diff
@@ -1080,116 +1078,128 @@ const WidgetConfig = ({ workspace, widget, onComplete, closeMenu }) => {
+                                    },
+                                    { value: 'object-marking.internal_id' },
+                                    {
+                                      value: 'kill-chain-phase.internal_id',
+                                    },
+                                    {
+                                      value: 'x_opencti_workflow_id',
+                                    },
+                                  ].map((attribute) => (
+                                    <MenuItem
+                                      key={attribute.value}
+                                      value={attribute.value}
+                                    >
+                                      {t_i18n(
+                                        capitalizeFirstLetter(
+                                          attribute.value,
+                                        ),
+                                      )}
+                                    </MenuItem>
+                                  ))}
+                                </Select>
+                              );
+                            }
+                            return <div/>;
+                          }}
+                        />
+                      </FormControl>
+                      )}
                     {dataSelection[i].perspective === 'entities'
-                                                && getCurrentSelectedEntityTypes(i).length === 0 && (
-                                                <TextField
-                                                  style={{
-                                                    flex: 1,
-                                                    marginRight:
-                                                                dataSelection[i].perspective === 'relationships'
-                                                                  ? 20
-                                                                  : 0,
-                                                  }}
-                                                  label={t_i18n('Field')}
-                                                  fullWidth={true}
-                                                  value={dataSelection[i].attribute}
-                                                  placeholder={t_i18n('Series attribute')}
-                                                  onChange={(event) => handleChangeDataValidationParameter(
-                                                    i,
-                                                    'attribute',
-                                                    event.target.value,
-                                                  )
-                                                        }
-                                                />
+                      && getCurrentSelectedEntityTypes(i).length === 0 && (
+                        <Select
```

Comment:
We don't have a label on this field anymore

Reply:
fixed